USAGE.md

```diff
@@ -19,17 +19,17 @@ This file describes how to run the project and develop against it.
   - If containers are not working, they may not be enabled, enabling them in docker settings and restarting the machine fixes this
```

Comment:
move the setup info from monorepo.md to here, make that file be about the monorepo and the process/goals of doing that